File: app/companions/app/src/main/java/com/niutao/restaurant/activity/MenuActivity.java
// MenuActivity.kt
class MenuActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu)
        val name = findViewById<EditText>(R.id.name)
        val description = findViewById<EditText>(R.id.description)
        val price = findViewById<EditText>(R.id.price)
        val image = findViewById<ImageView>(R.id.image)
        val upload = findViewById<Button>(R.id.upload)
        val save = findViewById<Button>(R.id.save)

        upload.setOnClickListener {
            // 上传照片逻辑
        }

        save.setOnClickListener {
            // 保存菜品逻辑
        }
    }
}

File: app/companions/app/src/main/java/com/example/ntcanteen/MenuActivity.kt
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.core.content.ContextCompat
import java.util.ArrayList

class MenuActivity : AppCompatActivity() {
    private lateinit var imageView: ImageView
    private lateinit var editText: EditText
    private lateinit var button: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu)

        imageView = findViewById(R.id.image_view)
        editText = findViewById(R.id.edit_text)
        button = findViewById(R.id.button)

        button.setOnClickListener {
            val menu = Menu(editText.text.toString(), imageView.drawable)
            val intent = Intent(this, CommentActivity::class.java)
            intent.putExtra("menu", menu)
            startActivity(intent)
        }
    }
}
